### PR TITLE
Improved phrasing

### PR DESCRIPTION
**Pull request by dmpop**
### PR creator: Description

Describe the overall goals of this pull request.

### PR creator: Are there any relevant issues/feature requests?

* bsc#...
* jsc#SLE-...

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [ ] SLE 15 SP4/openSUSE Leap 15.4 *(current `main`, no backport necessary)*
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
  - [ ] SLE 15 GA
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4
  - [ ] SLE 12 SP3

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Review by vmoutoussamy — APPROVED** [review]
Perfect, thanks a lot!

File: xml/containers-basics.xml [hunk 1]
```diff
@@ -66,16 +63,19 @@
...
+     x86_64 system host.
    </para>
   </listitem>
   <listitem>
    <para>
-    The security capabilities of containers are often overestimated and the overall security depends
-    on the host system. Containerized applications can be secured through &aa; or &selnx;
-    profiles. Container security would require implementing tools and policies to ensure that the
-    container infrastructure and applications are secure.
+    Containers in itself are no more secure than executing binaries outside of a
```
Comment (dcermak) [inline xml/containers-basics.xml:73]: This paragraph contains the word "container" in one form or another 4 times, I would suggest to rephrase this a bit as it reads a bit clunky.

**Review by dcermak — CHANGES_REQUESTED** [review]
I've highlighted a few points where I see room for improvement.

File: xml/containers-basics.xml [hunk 2]
```diff
@@ -12,17 +12,15 @@
...
-  hardware environment through a virtualization layer, Linux containers share and use directly the
-  host OS kernel.
-  This decrease the kernel overhead required for accessing the operating system resources an in turn
-  allow for better performance in general, compare to a generic Virtual Machine environment.
+  The  Linux kernel's namespaces and kernel control groups features (see <xref
+  linkend="cha-tuning-cgroups"/>) enable the container isolation from the
+  rest of the host system and other containers. Linux containers offer a
+  lightweight virtualization method to run multiple isolated environments
+  (containers) simultaneously on a single host. Unlike technologies like &xen;
+  or &kvm;, where the processor simulates a complete hardware environment
```
[the remark this answers is not in the document]
Comment (dmpop) [reply xml/containers-basics.xml:20]: Hm, I'm struggling to come up with something better. Any suggestions?

File: xml/containers-basics.xml [hunk 3]
```diff
@@ -66,16 +63,19 @@
...
    </para>
   </listitem>
   <listitem>
    <para>
-    The security capabilities of containers are often overestimated and the overall security depends
-    on the host system. Containerized applications can be secured through &aa; or &selnx;
-    profiles. Container security would require implementing tools and policies to ensure that the
-    container infrastructure and applications are secure.
+    Containers in itself are no more secure than executing binaries outside of a
+    container, and the overall security of containers depends on the host
```
[the remark this answers is not in the document]
Comment (dmpop) [reply xml/containers-basics.xml:74]: The comma is legit: it separates two independent sentences.

File: xml/containers-basics.xml (hunk 1 above)
Comment (dmpop) [reply xml/containers-basics.xml:73]: Ugh, good point!

File: xml/containers-basics.xml [hunk 4]
```diff
@@ -12,17 +12,15 @@
...
-  This decrease the kernel overhead required for accessing the operating system resources an in turn
-  allow for better performance in general, compare to a generic Virtual Machine environment.
+  The  Linux kernel's namespaces and kernel control groups features (see <xref
+  linkend="cha-tuning-cgroups"/>) enable the container isolation from the
+  rest of the host system and other containers. Linux containers offer a
+  lightweight virtualization method to run multiple isolated environments
+  (containers) simultaneously on a single host. Unlike technologies like &xen;
+  or &kvm;, where the processor simulates a complete hardware environment
+  through a virtualization layer, Linux containers share and use directly the
+  host OS kernel. This reduces the kernel overhead required for accessing the
```
Comment (dcermak) [inline xml/containers-basics.xml:22]:
```suggestion
  (containers) simultaneously on a single host. Unlike technologies like &xen;
  or &kvm;, where a full guest operating system is executed through a virtualization 
  layer, Linux containers share and directly use the host OS kernel. This reduces
  the overhead required for accessing the
```

File: xml/containers-basics.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (dcermak) [reply xml/containers-basics.xml:20]: How about this: https://github.com/SUSE/doc-sle/pull/1047/files/7ba107a271a74e5ccfbdf4211164f493becf5393#r778102022

**Pull request merged**